deps/media-playback/media-playback/media.c

```diff
@@ -762,7 +762,18 @@ static int interrupt_callback(void *data)
 static bool init_avformat(mp_media_t *m)
 {
 	AVInputFormat *format = NULL;
-
+	if (m->is_local_file) {
```

Comment:
Should we set `is_local_file` to false if we fail to open? I think the issue with RTMP is that the file is not local but the box is checked right?

Reply:
fopen can only tell us what the path is unopenable right now. It still can be a valid local path( for example a folder with file renamed or a flash drive disconnected). So it is not enough information to switch off is_local_file.